swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, condition = 0, fabric = "Unknown"):
+        super().__init__(id, condition)
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
```

Comment:
This works! But what might happen if the name of this class was suddenly changed? We'd have to remember to change it down here as well. What's one way we could always make sure `get_category` will dynamically return the class's _name_?